### ISSUE TITLE
Replace "Twitter" logos and alt text with "X"

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyrax#7066]
Hyrax links to X, but the logo and alternative text used is still the Twitter blue bird. This should be updated throughout the software.

![Image](https://github.com/user-attachments/assets/00a82a33-c778-4db6-9f23-9d550fa2f6c1)
